pom.xml

```diff
@@ -25,12 +26,13 @@
   <dependencyManagement>
     <dependencies>
       <dependency>
-        <groupId>commons-logging</groupId>
-        <artifactId>commons-logging</artifactId>
-        <version>1.1.1</version>
+          <groupId>org.slf4j</groupId>
+          <artifactId>slf4j-api</artifactId>
+          <scope>provided</scope>
```

Comment:
please make sure to match the tabs/spaces file indentation.

Reply:
Fixed